[2025-09-30T17:40:06.131+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-30T17:40:06.162+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag2.task_pwd scheduled__2025-09-30T17:39:00+00:00 [queued]>
[2025-09-30T17:40:06.173+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag2.task_pwd scheduled__2025-09-30T17:39:00+00:00 [queued]>
[2025-09-30T17:40:06.173+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-09-30T17:40:06.220+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): task_pwd> on 2025-09-30 17:39:00+00:00
[2025-09-30T17:40:06.234+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=269) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-09-30T17:40:06.237+0000] {standard_task_runner.py:63} INFO - Started process 271 to run task
[2025-09-30T17:40:06.236+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag2', 'task_pwd', 'scheduled__2025-09-30T17:39:00+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/dag2.py', '--cfg-path', '/tmp/tmps3m1lq_5']
[2025-09-30T17:40:06.240+0000] {standard_task_runner.py:91} INFO - Job 57: Subtask task_pwd
[2025-09-30T17:40:06.272+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-09-30T17:40:06.338+0000] {task_command.py:426} INFO - Running <TaskInstance: dag2.task_pwd scheduled__2025-09-30T17:39:00+00:00 [running]> on host 7cd44293b4c0
[2025-09-30T17:40:06.470+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag2' AIRFLOW_CTX_TASK_ID='task_pwd' AIRFLOW_CTX_EXECUTION_DATE='2025-09-30T17:39:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-30T17:39:00+00:00'
[2025-09-30T17:40:06.473+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-30T17:40:06.475+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-09-30T17:40:06.477+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'pwd']
[2025-09-30T17:40:06.495+0000] {subprocess.py:86} INFO - Output:
[2025-09-30T17:40:06.498+0000] {subprocess.py:93} INFO - /tmp/***tmp5943727s
[2025-09-30T17:40:06.499+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-09-30T17:40:06.500+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-30T17:40:06.538+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag2, task_id=task_pwd, run_id=scheduled__2025-09-30T17:39:00+00:00, execution_date=20250930T173900, start_date=20250930T174006, end_date=20250930T174006
[2025-09-30T17:40:06.612+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-09-30T17:40:06.635+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-30T17:40:06.637+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
